lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -48,6 +48,20 @@ my @excludedCurrencies = (
     'usd'
 );
 
+# Used to filter on queries of the form '1 <cryptocurrency>'
```

Comment:
Can you add the source of the top currencies to this comment `coinmarketcap.com` for future reference.